examples/example-with-encryption/README.md

```diff
@@ -0,0 +1,23 @@
+# Consul cluster with encryption example
+
+This folder contains an example of a Packer manifest that creates an AMI with the required certs for TLS validation, as well as installing an updated version of the `run-consul` script that accepts parameters for enabling RPC and/or gossip encryption.
```

Comment:
You should mention that this folder also contains Terraform configurations that can be used to deploy the AMI to AWS.